shreya_custom_loss_test.py

```diff
@@ -0,0 +1,74 @@
+from os import remove
+from tkinter.font import names
+
+# from numpy import error_message
+
+from ansys.motorcad.core import MotorCAD
+
+mc = MotorCAD(open_new_instance=False)
+
+# Global variables
+kNumCustomLossesInternal_Lab = "NumCustomLossesInternal_Lab"
+kCustomLoss_name_internal_lab = "CustomLoss_name_internal_lab"
+kCustomLoss_Function_Internal_Lab = "CustomLoss_Function_Internal_Lab"
+kCustomLoss_Type_Internal_Lab = "CustomLoss_Type_Internal_Lab"
+kCustomLoss_ThermalNode_Internal_Lab = "CustomLoss_ThermalNode_Internal_Lab"
+kNumCustomLossesExternal_Lab = "NumCustomLossesExternal_Lab"
+kCustomLoss_Name_External_Lab = "CustomLoss_Name_External_Lab"
+kCustomLoss_PowerFunction_External_Lab = "CustomLoss_PowerFunction_External_Lab"
+kCustomLoss_VoltageFunction_External_Lab = "CustomLoss_VoltageFunction_External_Lab"
+
+def add_internal_custom_loss(name, function, type, thermal_node):
+    type = type.capitalize()
+    if type not in ["Electrical", "Mechanical"]:
```

Comment:
This will work but I prefect removing
```
 type = type.capitalize()
```

and using

```suggestion
    if type.lower() not in ["electrical", "mechanical"]:
```

I think this is more robust

Reply:
As discussed  -leave a comment to explain this